version.json

```diff
@@ -1,5 +1,5 @@
 {
-  "version": "6.0.0-build.{height}",
+  "version": "6.0.0",
```

Comment:
```suggestion
  "version": "6.0.0-build.{height}",
```

Reply:
I dont understand the implication of this suggestion, could you elaborate?

---
In reply to: [260043392](https://github.com/windows-toolkit/Microsoft.Toolkit.Win32/pull/85#discussion_r260043392) [](ancestors = 260043392)

Reply:
This file specifies the version that the build process will use on the NuGet package. This "-build.{height}" will make sure that the build number is unaltered.
It's mostly just for cleanness, since this will most likely be overridden when we merge this on rel/6.0.0 with a preview2 or something like that.

Reply:
Thanks.

---
In reply to: [260487797](https://github.com/windows-toolkit/Microsoft.Toolkit.Win32/pull/85#discussion_r260487797) [](ancestors = 260487797)